lib/ui/analytics/analytics.js

```diff
@@ -23,7 +23,8 @@
       // convert the directive attributes into object with properties with sane defaults
       var properties = angular.extend(
         {
-          level: 'info'
+          level: 'info',
+          label: element.html()
```

Comment:
element.html()  handled most the use cases except if there was html within the parent, so element.text() is the correct approach. Made the change and pushed